notebook/static/tree/js/notebooklist.js

```diff
@@ -595,6 +597,20 @@ define([
             $('.delete-button').css('display', 'none');
         }
 
+        // View is visible when an item is renderable or downloadable
```

Comment:
As far as I have tested, it seems to work (Chrome). I had to create a local variable to assign the window to, otherwise I only got the one window. I haven't checked to see if there is a maximum number ... yet.